package.json

```diff
@@ -4,7 +4,7 @@
   "version": "1.47.1",
   "description": "A framework to support application development with Terra components",
   "engines": {
-    "node": ">=8.9.2 <11"
+    "node": ">=10 <15"
```

Comment:
Shouldn't this be `<14`?

Reply:
terra-application can also support Node 14 so we want to include 14. Just not 16 yet.